pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -399,6 +400,14 @@ class Build {
                                 additionalTestLabel += '&&'
                             }
                             additionalTestLabel += 'sw.tool.podman&&(sw.os.ubuntu.22||sw.os.rhel.8)'
+                        } else if (testType  == 'dev.jck') {
+                            if (buildConfig.TARGET_OS == "aix") {
+	                            if (additionalTestLabel != '') {
+	                                additionalTestLabel += '&&'
+	                            }  
+	                            additionalTestLabel += 'ci.geo.raleigh&&sw.tool.jckshare'
+	                            relatedNodeLabel = 'ci.geo.raleigh&&sw.tool.jckrefserver'
+	                        }
```

Comment:
I remembered that we have code to handle the platform that is not aix. Why the code is removed? In [jdk11u_pipeline_config.groovy](https://github.com/ibmruntimes/ci-jenkins-pipelines/pull/123/files#diff-cd68afc4fbcb6c7457170bc83490eb803243e4de6cfc3841de93d569cd475a74), some other platforms are enabled, what additionalTestLabel and relatedNodeLabel will they run?

Reply:
I see your comment in https://github.com/ibmruntimes/ci-jenkins-pipelines/pull/123#discussion_r1122052791. One node solution has not been delivered yet. These platforms should not be enabled until https://github.com/adoptium/aqa-tests/pull/4372 is merged.

Reply:
@llxia , Yes. https://github.com/ibmruntimes/ci-jenkins-pipelines/pull/123#discussion_r1122052791 has the answers. 
 
My understanding was that these changes will only be used to generate the dev.jck jobs, so it may not depend on https://github.com/adoptium/aqa-tests/pull/4372, plus, on the upside, we can have the multi-node solution enabled on AIX.

Reply:
> My understanding was that these changes will only be used to generate the dev.jck jobs


No, this change will also trigger the test jobs in our nightly/weekly runs.